### DISCUSSION TITLE
Google calendar

### DISCUSSION THREAD

**Discussion by lucasfth** [PhilipFlyvholm/learnit-plus-plus#33] [Ideas]
It seems to be possible to add multiple of your google calendars into a single iframe.
I am therefore thinking of trying to add a feature in the future where it allows you to show a google calendar instead of timeedit.

[What I have found so far](https://webapps.stackexchange.com/questions/119268/how-can-i-display-the-events-from-two-google-calendars-in-a-single-iframe).

**Comment by PhilipFlyvholm** [comment]
That sounds like a good solution. I have also thought about parsing the TimeEdit ical files, just like we do currently with student council events, instead of using an iframe to get a better calendar.
